plugins/social_network/social_network.cpp

```diff
@@ -282,6 +282,96 @@ namespace golos { namespace plugins { namespace social_network {
                 });
             }
         }
+
+        result_type operator()(const author_reward_operation& op) const {
+            if (!db.has_index<comment_reward_index>())
+                return;
+
+            const auto &comment = db.get_comment(op.author, op.permlink);
+
+            const auto& cr_idx = db.get_index<comment_reward_index>().indices().get<golos::plugins::social_network::by_comment>();
+            auto cr_itr = cr_idx.find(comment_id_type(comment.id));
+            if (cr_itr == cr_idx.end()) {
+                db.create<comment_reward_object>([&](golos::plugins::social_network::comment_reward_object& cr) {
```

Comment:
Rewards come here in order of send. So you can accumulate counters and update only on comment_payout_update_operation